src/network/mcpe/NetworkSession.php

```diff
@@ -213,6 +213,10 @@ public function getLogger() : \Logger{
 		return $this->logger;
 	}
 
+	public function getConnectedTime(): int {
+		return time() - $this->connectTime;
+	}
+
```

Comment:
Why not move this function directly to the `Player.php`, which would be more useful ?